src/api.py

```diff
@@ -1386,10 +1390,11 @@ def HandleDeleteAndVacuum(self, params):
         return None
 
     def HandleShutdown(self, params):
-        """Handle a request to huutdown the client"""
+        """Handle a request to shutdown the node"""
 
         if not params:
-            shutdown.doCleanShutdown()
+            # backward compatible trick because False == 0 is True
+            state.shutdown = False
```

Comment:
I think this shouldn't change.

Reply:
I reused this commit from `api` branch because I needed `TestAPIProto`.  Maybe I'll make a separate PR for shutdown to show that it never returns.